project/OSGi.scala

```diff
@@ -9,14 +9,15 @@ import com.typesafe.sbt.osgi.SbtOsgi._
 import com.typesafe.sbt.osgi.SbtOsgi.autoImport._
 import sbt._
 import sbt.Keys._
+import net.bzzt.reproduciblebuilds.ReproducibleBuildsPlugin
 
 object OSGi {
 
   // The included osgiSettings that creates bundles also publish the jar files
   // in the .../bundles directory which makes testing locally published artifacts
   // a pain. Create bundles but publish them to the normal .../jars directory.
   def osgiSettings = defaultOsgiSettings ++ Seq(
-    Compile / packageBin := OsgiKeys.bundle.value,
+    Compile / packageBin := ReproducibleBuildsPlugin.postProcessJar(OsgiKeys.bundle.value),
```

Comment:
Isn't it weird to sneak it in here in the OSGi config or does it have to be here to know that it happens after the OSGi bundle task

Reply:
`ReproducibleBuildsPlugin.postProcessJar` is normally injected into `Compile/packageBin` automatically by the reproducible-builds plugin. Here we replace the `Compile/packageBin` task wholesale with the OSGi bundle, so we need to 're-introduce' the `ReproducibleBuildsPlugin.postProcessJar` explicitly. We could do it elsewhere but I think that would make it even harder to follow. I'll add a comment explaining this though.

Reply:
Couldn't it be it's own autoplugin and just express the ordering by depending on the OSGi autoplugin though?